src/main/java/core/basesyntax/interfaces/InfoPrinter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.interfaces;
+
+public interface InfoPrinter {
+    String drawFigure();

```

Comment:
```suggestion
    String getInfo();
```
we can get into not about figures only, right?

Reply:
I'm pretty sure that in this particular case, we are getting info only about this list of 5 figures. But if to take to attention our interface's name, then probably better to change the method's name to String getInfo();